### PR TITLE
Don't attempt to retrieve frameCollections on a destroyed graphic.

### PR DESCRIPTION
**Pull request by EliteMasterEric**
This fixes a crash bug I encountered while developing Funkin'.

### CODE REVIEW

**Comment by Geokureli** [comment]
I'd like to see if previous destroy checks have covered this, do you know a solid way to reproduce this?

**Comment by EliteMasterEric** [comment]
I think something was calling getFrameCollections manually. I don't recall exactly what I was working on but I think it was FlxAnimate?

File: flixel/graphics/FlxGraphic.hx [hunk 1]
```diff
@@ -514,6 +514,8 @@ class FlxGraphic implements IFlxDestroyable
 	 */
 	public inline function getFramesCollections(type:FlxFrameCollectionType):Array<Dynamic>
 	{
+		if (this.isDestroyed) return [];
+
```
Comment (Geokureli) [inline flixel/graphics/FlxGraphic.hx:518]:
```suggestion
		if (this.isDestroyed)
		{
			FlxG.log.warn('Invalid call to getFramesCollections on a destroyed graphic');
			return [];
		}
		
```

**Review by Geokureli — CHANGES_REQUESTED** [review]
see suggestions

File: flixel/graphics/FlxGraphic.hx (hunk 1 above)
Comment (EliteMasterEric) [reply flixel/graphics/FlxGraphic.hx:518]: Seems good! I think I may have had a warning in a previous iteration of this.

**Comment by Geokureli** [comment]
Thanks!

**Pull request merged**